dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -449,6 +457,204 @@ public synchronized void update(LogProviderConfig config) {
+                LogTraceData.MESSAGE_NAMES1_1[1] = entry.getValue();
+            } else if (LogFieldConstants.IBM_THREADID.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[2] = entry.getKey();
+            } else if (LogFieldConstants.MODULE.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[3] = entry.getValue();
+            } else if (LogFieldConstants.SEVERITY.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[4] = entry.getValue();
+            } else if (LogFieldConstants.LOGLEVEL.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[5] = entry.getValue();
+            } else if (LogFieldConstants.IBM_METHODNAME.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[6] = entry.getKey();
+            } else if (LogFieldConstants.IBM_CLASSNAME.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[7] = entry.getValue();
+            } else if (LogFieldConstants.LEVELVALUE.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[8] = entry.getValue();
+            } else if (LogFieldConstants.THREADNAME.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[9] = entry.getValue();
+            } else if (LogFieldConstants.CORRELATION_ID.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[10] = entry.getKey();
+            } else if (LogFieldConstants.ORG.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[11] = entry.getValue();
+            } else if (LogFieldConstants.PRODUCT.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[12] = entry.getValue();
+            } else if (LogFieldConstants.COMPONENT.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[13] = entry.getValue();
+            } else if (LogFieldConstants.IBM_SEQUENCE.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[14] = entry.getKey();
+            } else if (LogFieldConstants.THROWABLE.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[15] = entry.getValue();
+            } else if (LogFieldConstants.THROWABLE_LOCALIZED.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[16] = entry.getKey();
+            } else if (LogFieldConstants.MESSAGE.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[17] = entry.getValue();
+            } else if (LogFieldConstants.FORMATTEDMSG.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[18] = entry.getValue();
+            } else if (LogFieldConstants.EXTENSIONS_KVPL.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[19] = entry.getValue();
+            } else if (LogFieldConstants.OBJECT_ID.equals(entry.getKey())) {
+                LogTraceData.MESSAGE_NAMES1_1[20] = entry.getValue();
+            }
```

Comment:
need a final else condition to print a warning that they provided a key that doesn't match one of the known keys.